data-access-layer/eventmesh/ApiServerClient.js

```diff
@@ -130,6 +199,15 @@ class ApiServerClient {
         }));
   }
 
+  patchResourceStatus(resourceGroup, resourceType, resourceId, statusDelta) {
```

Comment:
why is the last param called `delta` its just `status` right?

Reply:
if we put status it may mean that the value sent will replace the current status, but that wont be the case.
the `statusDelta` is patched and not updated.
The existing status values are retained and only the changed keys are updated.